FAQ.md

```diff
@@ -1,202 +1,122 @@
+We do not offer job services, however, completing TOP means you have the skills and qualifications to find a job, see <#669547324707569665> and the getting hired section of the curriculum. <#1013804065169346631> are also a good source of motivation.
 
 **Does TOP give certificates?**
 
 No, certificates do not have much value outside of very small niches in IT, usually not programming. After TOP, your portfolio should speak for itself
 
+**When should I use ModMail?**
+
+If you see anything in the server that you believe is against our community rules, we encourage you to send a DM to <@575252669443211264> and to not address the issue yourself. You can also send a DM to <@575252669443211264> if you're not sure if you're allowed to share something in our channels.
+
+**When should I NOT use Modmail?** 
+
+Please don't use <@575252669443211264> for personal TOP advice, programming help or casual chats with the team.
+
 **Can I post my affiliate links, self promote, advertise my server/product here?**
 
-Almost always no. Do not assume you have permission. We may entertain things that are not designed to make money or take away from the mission of TOP.
+Almost always no. Do not assume you have permission. We may entertain things that are not designed to make money or take away from the mission of TOP. Please ask us in a DM to <@575252669443211264>.
 
 **How long does TOP take?**
 
-There is no data on this information, between 3 months and 3 years seems to be the range. Read this for more info: 
-
-> For anybody worried about "how long it takes", consider these things:
->
-> It may take you longer than others to grasp concepts, or it may take you less time. This doesn't mean you're smarter or dumber than others, it means you've had differing life experiences that may or may not have prepared you for learning this stuff. Someone who grew up around an engineer may have some advantages over someone who didn't, but it doesn't mean you can't learn those skills.
->
-> The Odin Project isn't like college or university, TOP is self paced and allows you to get a really solid grasp of concepts before moving on. In school, you're forced to keep up or you will fail. The difference here is that coming into TOP, you're not expected to have much knowledge, there's no prerequisites. We've had people be successful coming through here that only knew how to check their email with a computer, we've also seen success from computer science degree holders. Treating TOP like a static timeline is ignorant simply because you don't know what you don't know yet, and that's OK! There's no due-dates on things in The Odin Project so you can spend the time to do it right and discuss the topics.
+There is no data on this information, between 3 months and 3 years seems to be the range. More info [here](https://discord.com/channels/505093832157691914/505093832157691916/765633002393829389).
```

Comment:
I'm not sure I have done this correctly. I updated the suggestion to `Read this [great message about the time it may or may not take]` and now I'm not sure that removes your suggestion